.github/dependabot.yml

```diff
@@ -16,10 +16,14 @@ updates:
     - "m1l4n54v1c"
```

Comment:
I don't think this change belongs in your PR, right? :thinking: 
It should be part of Framework/master for some time now, actually.

Reply:
Hmm, it's actually not on master right now... I'll cherry-pick and merge it into master so that it won't pop up in this draft anymore.